lib/travis/addons/model/user.rb

```diff
@@ -9,11 +9,11 @@ def with_github_token
     end
 
     def with_permissions(permissions)
-      where(:permissions => permissions).includes(:permissions)
+      where(permissions:).includes(:permissions)
```

Comment:
@gbarc-dt it shouldn't it be `where(permissions: permissions).includes(:permissions)` ?

Reply:
never mind.